packages/core/core/.eslintignore

```diff
@@ -0,0 +1,2 @@
+src/**/*.ts
```

Comment:
Was eslint actually checking the `.ts` files. If so, that's very interesting; I though it just ignored anything that's not a `.js` file.

Reply:
This is also a leftover. Chances are, I confused tsc, tslint, eslint + babel parsing errors as I went through this. Will remove this.